pages/docs/concepts/asyncapi-document/securing-operations.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Securing Operations'
+weight: 120
+---
+
+In an AsyncAPI document, the concept of server security implies that the security measures defined at the server level apply to all operations within all channels by default. 
+
+## Features of Security in AsyncAPI Documents
+
+- The security requirements specified at the server level are enforced consistently across the entire asyncapi document.
+
+There may be situations where certain operations within specific channels require different security measures than the default server-level settings. To accommodate for such scenarios, AsyncAPI document allows you to use the `security` property at the `operation` level.
+
+By using the `security` property at the `operation` level, you can override the default server-level security and define unique security requirements for individual operations. Thereby also giving you the flexibility to tailor the security measures to the specific needs of each operation, even if they differ from the broader server-level settings.
+
+## Specifying Security Requirement 
+
+To specify different security requirements for a specific operation, you can include the `security` property within the operation's definition. 
+The security property is an array where you can define one or more security requirement objects.
+
+For example, let's say you have an AsyncAPI document with a channel called users and two operations within that channel: createUser and getUser. 
+The server-level security is set to use API key authentication for all operations within all channels. 
+However, you want to enforce OAuth2 authentication specifically for the getUser operation.
+
+You can achieve this by including the security property at the operation level, as shown in the following example:
+
+```yaml
```

Comment:
this is v2 example

Reply:
replaced with v3 example

![image](https://github.com/asyncapi/website/assets/78029145/ffb685c2-5151-434f-a239-a10daebf7757)